portfolio/src/main/webapp/script.js

```diff
@@ -50,24 +50,69 @@ async function showSongLyrics() {
 })(jQuery);
 
 /** Script for time */
-function GetClock(){
-    var d=new Date();
-    var nhour=d.getHours(),nmin=d.getMinutes(),ap;
-    if(nhour==0){ap=" AM";nhour=12;}
-    else if(nhour<12){ap=" AM";}
-    else if(nhour==12){ap=" PM";}
-    else if(nhour>12){ap=" PM";nhour-=12;}
-    
-    if(nmin<=9) nmin="0"+nmin;
-    
-    document.getElementById('clockbox').innerHTML=""+nhour+":"+nmin+ap+"";
+function getClock() {
+    var date = new Date();
+    var hour = date.getHours();
+    var min = date.getMinutes();
+    var midday = "AM";
+    midday = (hour >= 12) ? "PM" : "AM";
+    hour = (hour == 0) ? 12 : ((hour > 12) ? (hour - 12): hour);
+    hour = updateTime(hour);
+    min = updateTime(min);
+    const clockBock = document.getElementById('clockbox');
+    clockBock.innerHTML = hour + ":" + min + " " + midday;
+}
+
+function updateTime(k) { //appending 0 before time elements if less than 10
+    if (k < 10) {
+        return "0" + k;
+    }
+    else {
+        return k;
+    }
 }
- 
 window.onload=function(){
-    GetClock();
-    setInterval(GetClock,1000);
+    this.batteryIcon();
+    this.volumeIcon();
+    getClock();
+    setInterval(getClock,1000);
 }
 
+
+
+/*
+Random Image Script- By JavaScript Kit (http://www.javascriptkit.com)
+Over 400+ free JavaScripts here!
+Keep this notice intact please
+*/
+function volumeIcon(){
+    var tbvol=new Array();
+                            
+    tbvol[1]="up";
+    tbvol[2]="down";
+    tbvol[3]="off";
+                            
+    var level=Math.floor(Math.random()*tbvol.length+1); //generate random volume level
+    document.getElementsByClassName('tbvolume')[0].innerHTML = '<i class="fa fa-volume-'+tbvol[level]+level+' "></i>';
```

Comment:
Update this line to:
document.getElementsByClassName('tbvolume')[0].innerHTML = '<i class="fa fa-volume-'+tbvol[level]+' "></i>';

I think the issue is you are adding a level here & you need to call the functions somewhere in the script.